docs/Operators.md

```diff
@@ -7510,14 +7510,14 @@ expect(node, inputs=[x], outputs=[y],
   
       graph body-net (
         %i[INT32, scalar]
-        %keepgoing[BOOL, scalar]
-        %b[INT32, scalar]
+        %keepgoing_in[BOOL, scalar]
+        %b_in[INT32, scalar]
       ) {
-        %my_local = Add(%a, %b)
-        %b_out = Sub(%a, %b)
+        %my_local = Add(%a, %b_in)
+        %b_out = Sub(%a, %b_in)
```

Comment:
```suggestion
        // The value of b_out is used as the value of b_in in the next iteration. Note that it doesn't affect the value of the b_in passed into this operator.
        %b_out = Sub(%a, %b_in)
```

Reply:
I added a few more comments to address the two points you mention.